pkg/controller.v1/pytorch/master.go

```diff
@@ -14,6 +14,9 @@ var (
 	onceMaster      sync.Once
 	EnvMasterPort   = "MASTER_PORT"
 	EnvMasterAddr   = "MASTER_ADDR"
+
+	PETMasterPort = "PET_MASTER_PORT"
```

Comment:
You need this to auto populate `--master_port` and `--master_addr` in torchrun cmd arguments? Thinking, if there is a way to avoid duplication?

Reply:
we can create an issue to cleanup in future API release

Reply:
For now, I can not find a way to support the 3 way in the same time without duplicating those. Maybe for a future deprecation.